actioncable/lib/action_cable/channel/base.rb

```diff
@@ -32,7 +32,7 @@ module Channel
     #
     # == Action processing
     #
-    # Unlike Action Controllers, channels do not follow a REST constraint form for its actions. It's a remote-procedure call model. You can
+    # Unlike ActionControllers, channels do not follow a REST constraint form for their actions. It's a remote-procedure call model. You can
```

Comment:
I don't think we've ever used ActionControllers (with or without the space) to mean "instances of AC::Base". It feels awkward to me, but I don't have a concrete suggestion for an alternative. :confused: